spring-thymeleaf-2/src/main/resources/templates/mvcdata/email.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+	  xmlns:th="http://www.thymeleaf.org">
+<head>
+</head>
+<h1>Subject</h1>
+<p th:text="${emailsubject}">Model attributes</p>
+</br>
+<h1>Language</h1>
+<p th:text="${param.locale}">Request parameters</p>
+</br>
+<h1>Content</h1>
+<p th:text="${session.emailcontent}">Session attributes</p>
+</br>
+<h1>Recipient name</h1>
+<p th:text="${#servletContext.getAttribute('emailreceiver')}">Servlet context attributes</p>
+</br>
+<h1>Email address</h1>
+<p th:text="${@emailAddress}">Bean data</p>
+<body>
```

Comment:
Thsse empty `<body>` and `<head>` tags seem odd. Let's remember to clean this up before merging.

Reply:
I removed this whole file, it was no longer needed